nh_eobs_mobile/static/dev/coffee/src/nhmobilepatient.coffee

```diff
@@ -129,6 +134,10 @@ class NHMobilePatient extends NHMobile
         chartFunc(self, serverData)
       if validTable
         tableFunc(self, serverData)
+      if activeTab is "#graph-content"
+        tableEl.style.display = "none"
+      else
+        graphContent.style.display = "none"
```

Comment:
Yeah it draws both at the same time when the dropdown is changed. I contemplated doing this on tab load instead but that would mean having to contact the server on tab press so decided to instead contact the server just the once per observation change